zetaclient/chains/evm/rpc/rpc.go

```diff
@@ -0,0 +1,52 @@
+package rpc
+
+import (
+	"context"
+
+	ethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/pkg/errors"
+
+	"github.com/zeta-chain/zetacore/zetaclient/chains/interfaces"
+)
+
+// IsTxConfirmed checks if the transaction is confirmed with given confirmations
+func IsTxConfirmed(
```

Comment:
Is it possible to remove "evm json rpc" and use only `interfaces.EVMRPCClient`?

Reply:
This can be removed once we can upgrade the dependency of `github.com/ethereum/go-ethereum` to latest version. It's now being used to observe Ethereum inbound `Blob` transactions.

Reply:
Are there any blocker for upgrading go-ethereum?

Reply:
https://github.com/zeta-chain/ethermint/issues/86

Reply:
The blocker is tracked in above link.